hack/snapshot_to_catalog.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+usage() {
+  echo "Usage: $0 <snapshot-ref> <catalog-file> <bundle-tag>"
+  echo "  snapshot-ref: the snapshot reference to use, example: ols-bnxm2"
+  echo "  catalog-file: the catalog index file to update, example: lightspeed-catalog-4.16/index.yaml"
+  exit 1
+}
+
+if [ $# == 0 ]; then
+  usage
+fi
+
+SNAPSHOT_REF=${1:-"ols-bnxm2"}
```

Comment:
@raptorsun Do we want a default option ? We want to force the user to provide snapshot ref .

Reply:
now updated: snapshot is required, only catalog file is default to the 4.16 catalog file